parser/feed/src/main/java/de/danoeh/antennapod/parser/transcript/PodcastIndexJsonTranscriptParser.java

```diff
@@ -0,0 +1,77 @@
+package de.danoeh.antennapod.parser.transcript;
+
+import android.util.Log;
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.jsoup.internal.StringUtil;
+
+import de.danoeh.antennapod.model.feed.Transcript;
+import de.danoeh.antennapod.model.feed.TranscriptSegment;
+
+public class PodcastIndexJsonTranscriptParser {
+    static String TAG = "PodcastIndexJsonTranscriptParser";
+    // merge short segments together to form a span of 1 second
+    static long minSpan = 1000L;
+
+    public static Transcript parse(String jsonStr) {
+        try {
+            Transcript transcript = new Transcript();
+            long startTime = 0L;
```

Comment:
Your other code uses -1 here, so should this be -1? https://github.com/tonytamsf/AntennaPod/pull/6/files#diff-14d98525c5df8d24fa975f628614e462438e8ccb86a08dab5f5bafabb2ef09cfR418

Reply:
Yup, -1L makes more sense